utam-compiler/src/test/java/utam/compiler/translator/JsonCompilerConfigTests.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root
+ * or https://opensource.org/licenses/MIT
+ */
+package utam.compiler.translator;
+
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.anEmptyMap;
+import static org.hamcrest.Matchers.emptyIterable;
+import static org.hamcrest.Matchers.emptyString;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.testng.Assert.expectThrows;
+import static utam.compiler.translator.JsonCompilerConfig.ERR_READING_COMPILER_CONFIG;
+import static utam.compiler.translator.JsonCompilerConfig.ERR_TARGET_NOT_SET;
+import static utam.compiler.translator.JsonCompilerConfig.Module.DEFAULT_JSON_FILE_MASK_REGEX;
+import static utam.compiler.translator.JsonCompilerConfig.Namespace.ERR_DUPLICATE_MAPPING;
+import static utam.compiler.translator.JsonCompilerConfig.Profile.ERR_DUPLICATE_PROFILE_DIFF_VALUES;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.File;
+import java.io.IOException;
+import java.util.Collection;
+import org.hamcrest.CoreMatchers;
+import org.testng.annotations.Test;
+import utam.compiler.UtamCompilationError;
+import utam.compiler.translator.JsonCompilerConfig.Module;
+import utam.compiler.translator.JsonCompilerConfig.Namespace;
+import utam.compiler.translator.JsonCompilerConfig.Profile;
+import utam.compiler.translator.JsonCompilerConfig.Target;
+import utam.core.declarative.translator.ProfileConfiguration;
+import utam.core.declarative.translator.TranslatorSourceConfig;
+import utam.core.declarative.translator.UnitTestRunner;
+import utam.core.framework.consumer.UtamError;
+import utam.core.framework.context.StringValueProfile;
+
+/**
+ * tests for json config
+ *
+ * @author elizaveta.ivanova
+ * @since 234
+ */
+public class JsonCompilerConfigTests {
```

Comment:
this is not new file, simply renamed JsonBasedCompilerConfigTests.java

Reply:
Understood. Approved for this file.